app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
```

Comment:
I observe that the `:recoverable` feature is enabled in the user model (user.rb file). 

Despite this configuration, I encountered difficulty resetting their passwords in the application, leading to an error.

![image](https://github.com/obibaadoma/Blog_App/assets/78251927/2220718a-2146-4981-8d91-3c35c09181f2)


Password recovery is crucial for assisting users in regaining access to their accounts or systems in cases of forgotten or lost passwords.

Passwords are essential for authentication and protecting user data and privacy, so password recovery is vital for both user satisfaction and security.

Please fix this error 🛠 🤗